modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk/query_processor.clj

```diff
@@ -194,6 +194,16 @@
   [driver [_ value]]
   (h2x/maybe-cast "BIGINT" (sql.qp/->honeysql driver value)))
 
+(defmethod sql.qp/->honeysql [:bigquery-cloud-sdk :split-part]
+  [driver [_ text divider position]]
+  [:coalesce
```

Comment:
Would also be good in #55941 to expand the tests cases, e.g. to tests this `coalesce` handling, but also just testing that you get the expected output for specific known inputs.